content/ecosystem/index.html

```diff
@@ -9,7 +9,7 @@
 		<link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,700;1,400&family=Red+Hat+Display:ital,wght@0,400;0,500;0,700;1,400;1,500&display=swap" rel="stylesheet">
 		<script src="/init.js"></script>
 
-<script async defer data-domain="cassandra.apache.org" src="https://plausible.cassandra.apache.org/js/plausible.js"></script></head>
+	</head>
```

Comment:
this line needs to stay. it's provides the tracking we see at https://plausible.cassandra.apache.org/cassandra.apache.org?period=realtime

Reply:
I really shouldn't have just accepted what was supplied and instead edit the existing file. Let me redo the PR. Thanks.

Reply:
the plausible tracking is new. i've seen @jtlevy add it to files, but for these custom and temporary html files it is easy to forgot to add this line.